omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -14,6 +14,15 @@ def ownership
     {"owner" => owner, "group" => group}
   end
 
+  def external_rabbitmq
```

Comment:
Maybe just name this rabbitmq_configuration since the user doesn't care whether it is external or not?